support/run-server

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# This script is a wrapper around running a server and its migrations. It's
+# necessary because forego doesn't let you run multiple commands.
+
+set -e
+
+if [ "$#" -lt "1" ] || [ "$#" -gt "2" ]; then
+  echo "Usage: run-server <SERVER> [CONFIG_PATH]"
+  echo "ex: run-server jobsrv /tmp/jobsrv.toml"
+  exit 1
+fi
+
+# base_dir is the root of the habitat project, which eliminates the need
+# to hardcode absolute paths.
+base_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+migrate="$base_dir/target/debug/bldr-$1 migrate"
+start="$base_dir/target/debug/bldr-$1 start"
+
+if [ "$#" -eq "2" ]; then
+  migrate+=" --config $2"
+  start+=" --config $2"
+fi
+
+eval $migrate
```

Comment:
You'll need to drop this first `eval` which turns this Bash process into the Rust `$base_dir/target/debug/bldr-$1 migrate …` process (meaning that the next line would never be called as it isn't a Bash program anymore).

Not a bad idea to just wrap these calls in a script to call `* migrate` and then `* start`. Keeps the server code pretty clean!

Reply:
I thought that's what `exec` did, not `eval`.  This works in my local dev env, in that it starts up, migrations run, and the server continues to run afterwards.  If using `eval` is a bad practice, though, I'm happy to do something else.  I just wanted a way to say "execute the command that's contained inside this variable" and googling led me to `eval`.

Reply:
OMG, I misread this, sorry! I thought I read `exec` not `eval`. Ignore me!